dependencies.gradle

```diff
@@ -0,0 +1,13 @@
+// These dependencies have to be in lib/build.gradle for compilation _and_
+// in java/build.gradle and android/build.gradle for maven.
+dependencies {
+  compile 'org.msgpack:msgpack-core:0.8.2'
+  compile 'org.java-websocket:Java-WebSocket:1.3.1'
```

Comment:
Aren't we using the forked version of this? I thought we had done a rename of the package as well as the groupId.